components/google-cloud/containers/aiplatfrom/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Copyright 2021 Google LLC. All Rights Reserved.
```

Comment:
Should this file be in `aiplatform/containers` instead?

Reply:
Good catch!  done.